backend/python/app/services/interfaces/comment_service.py

```diff
@@ -17,6 +17,17 @@ def create_comment(self, comment):
         """
         pass
 
+    @abstractmethod
+    def get_comments_by_story_translation(self, story_translation_id):
+        """Return a list of Comments from a given StoryTranslation
+
+        :param story_translation_id: story translation id
+        :return: list of comments
+        :rtype: list of CommentResponseDTO
+        :raises Exception: if id of comment to be updated does not exist
```

Comment:
could we get exceptions if something about the database fails?

Reply:
yep, all service function would throw exceptions in the event of a database fails (though we have discussed that not all of them are gracefully handeled)